src/graph.js

```diff
@@ -60,6 +61,19 @@ module.exports = (function () {
         return this;
     };
 
+    Graph.prototype.dontRender = function() {
+        this._renderNow = false;
+
+        return this;
+    };
+
+    Graph.prototype.doRender = function() {
```

Comment:
I gave it some thought, and my rationale was enabling the user to say, e.g. "I want to do some extra shenanigans with the data, or the formatting, or something, so would you ~~hold my beer and watch this~~ wait with the rendering for a while and let me just add what I need", or maybe "the data's still pouring in through an unreliable piecemeal async source", but I admit, this might be the case of me optimizing prematurely.

I mean, it's your call, if you don't want them at all, I can just use the `_renderNow` variable which IS privatized, and nix the functions altogether.